From efault@gmx.de  Fri Jul 31 09:03:22 2009
From: Mike Galbraith <efault@gmx.de>
Date: Fri, 31 Jul 2009 07:52:58 +0200
Subject: Staging: rt2870sta interface names
To: devel <devel@driverdev.osuosl.org>
Cc: Greg KH <greg@kroah.com>
Message-ID: <1249019578.21447.13.camel@marge.simson.net>


The rest of the known universe prefers wlanN for wireless interface names, to
the point that some distro configuration tools, such as opensuse's Yast, don't
even allow the user to enter a name, and simply pre-select "wlan".

Signed-off-by: Mike Galbraith <efault@gmx.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2860/rt_linux.c    |    2 +-
 drivers/staging/rt2860/rt_main_dev.c |    4 ++--
 drivers/staging/rt2860/rt_profile.c  |    6 +++---
 3 files changed, 6 insertions(+), 6 deletions(-)

--- a/drivers/staging/rt2860/rt_linux.c
+++ b/drivers/staging/rt2860/rt_linux.c
@@ -776,7 +776,7 @@ VOID RTMPSendWirelessEvent(
 		if (pAddr)
 			pBufPtr += sprintf(pBufPtr, "(RT2860) STA(%02x:%02x:%02x:%02x:%02x:%02x) ", PRINT_MAC(pAddr));
 		else if (BssIdx < MAX_MBSSID_NUM)
-			pBufPtr += sprintf(pBufPtr, "(RT2860) BSS(ra%d) ", BssIdx);
+			pBufPtr += sprintf(pBufPtr, "(RT2860) BSS(wlan%d) ", BssIdx);
 		else
 			pBufPtr += sprintf(pBufPtr, "(RT2860) ");
 
--- a/drivers/staging/rt2860/rt_main_dev.c
+++ b/drivers/staging/rt2860/rt_main_dev.c
@@ -710,7 +710,7 @@ static NDIS_STATUS rt_ieee80211_if_setup
 	// find available device name
 	for (i = 0; i < 8; i++)
 	{
-		sprintf(slot_name, "ra%d", i);
+		sprintf(slot_name, "wlan%d", i);
 
 		device = dev_get_by_name(dev_net(dev), slot_name);
 		if (device != NULL)
@@ -727,7 +727,7 @@ static NDIS_STATUS rt_ieee80211_if_setup
 	}
 	else
 	{
-		sprintf(dev->name, "ra%d", i);
+		sprintf(dev->name, "wlan%d", i);
 		Status = NDIS_STATUS_SUCCESS;
 	}
 
--- a/drivers/staging/rt2860/rt_profile.c
+++ b/drivers/staging/rt2860/rt_profile.c
@@ -710,7 +710,7 @@ static int rtmp_parse_key_buffer_from_fi
 				CipherAlg = CIPHER_WEP128;
 			pAd->SharedKey[i][idx].CipherAlg = CipherAlg;
 
-			DBGPRINT(RT_DEBUG_TRACE, ("I/F(ra%d) Key%dStr=%s and type=%s\n", i, idx+1, keybuff, (KeyType == 0) ? "Hex":"Ascii"));
+			DBGPRINT(RT_DEBUG_TRACE, ("I/F(wlan%d) Key%dStr=%s and type=%s\n", i, idx+1, keybuff, (KeyType == 0) ? "Hex":"Ascii"));
 			return 1;
 		}
 		else
@@ -731,12 +731,12 @@ static int rtmp_parse_key_buffer_from_fi
 				CipherAlg = CIPHER_WEP128;
 			pAd->SharedKey[i][idx].CipherAlg = CipherAlg;
 
-			DBGPRINT(RT_DEBUG_TRACE, ("I/F(ra%d) Key%dStr=%s and type=%s\n", i, idx+1, keybuff, (KeyType == 0) ? "Hex":"Ascii"));
+			DBGPRINT(RT_DEBUG_TRACE, ("I/F(wlan%d) Key%dStr=%s and type=%s\n", i, idx+1, keybuff, (KeyType == 0) ? "Hex":"Ascii"));
 			return 1;
 		}
 		else
 		{//Invalid key length
-			DBGPRINT(RT_DEBUG_ERROR, ("I/F(ra%d) Key%dStr is Invalid key length! KeyLen = %ld!\n", i, idx+1, KeyLen));
+			DBGPRINT(RT_DEBUG_ERROR, ("I/F(wlan%d) Key%dStr is Invalid key length! KeyLen = %ld!\n", i, idx+1, KeyLen));
 			return 0;
 		}
 	}
